indico/modules/events/abstracts/forms.py

```diff
@@ -61,3 +63,54 @@ class AbstractSubmissionSettingsForm(IndicoForm):
     authorized_submitters = PrincipalListField(_("Authorized submitters"),
                                                description=_("These users may always submit abstracts, even outside "
                                                              "the regular submission period."))
+
+
+class EmailTemplateRuleFormEdit(IndicoForm):
+    """Form for editing a new e-mail template."""
+
+    title = TextField(_("Title"), [DataRequired()])
+    rules = EmailRuleListField(_("Rules"), [DataRequired()])
+    stop_on_match = BooleanField(_("Stop on match"), [DataRequired()], widget=SwitchWidget(),
+                                 description=_("Do not evaluate any other rules once this one matches."),
+                                 default=True)
```

Comment:
I'd put the `default=True` in the first line (so far we always have the `description` kwarg as the last one)